graphql_compiler/tests/test_emit_output.py

```diff
@@ -289,3 +294,239 @@ def test_datetime_output_representation(self):
 
         received_match = emit_gremlin.emit_code_from_ir(ir_blocks, None)
         compare_gremlin(self, expected_gremlin, received_match)
+
+
+class EmitCypherTests(unittest.TestCase):
+    def setUp(self):
+        """Disable max diff limits for all tests."""
+        self.maxDiff = None
+
+    def test_simple_immediate_output(self):
+        # corresponds to:
+        # graphql_string = '''{
+        #     Foo {
+        #         name @output(out_name: "foo_name")
+        #     }
+        # }'''
+
+        base_location = Location(('Foo',))
+        base_name_location = base_location.navigate_to_field('name')
+        base_location_info = LocationInfo(
+            parent_location=None,
+            type=GraphQLObjectType(name='Foo', fields={'name': GraphQLString}),
```

Comment:
This is an example of a place where having the test helpers available would help -- you wouldn't have to construct a new type object, which reduces the setup footprint of the test and makes it less likely that the test setup itself breaks in the future.